### PR TITLE
new instrument cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20240214__CC_taarray.ext [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+0.003480,0.003189,0.002930,0.003058,0.002944,0.002857,0.002764,0.002685,0.002828,0.002176,0.001978,0.002191,0.002293,0.002149,0.002112,0.001731,0.001591,0.001290,0.001063,0.000990,0.000895,0.000709,0.000345,0.000163,0.000193,0.000000,-0.000013,-0.000253,-0.000328,-0.000377,-0.000542,-0.000664,-0.000815,-0.001062,-0.001124
+0.003278,0.003073,0.002840,0.002976,0.002869,0.002725,0.002712,0.002658,0.002762,0.002118,0.001968,0.002208,0.002272,0.002228,0.002164,0.001843,0.001668,0.001360,0.001179,0.001102,0.001078,0.000758,0.000397,0.000195,0.000296,0.000000,-0.000001,-0.000202,-0.000194,-0.000297,-0.000384,-0.000631,-0.000691,-0.000955,-0.000989
+0.003334,0.003031,0.002791,0.002926,0.002791,0.002753,0.002676,0.002635,0.002775,0.002097,0.001825,0.002133,0.002195,0.002178,0.002102,0.001788,0.001559,0.001322,0.001002,0.000981,0.000879,0.000634,0.000242,0.000070,0.000136,0.000000,-0.000113,-0.000349,-0.000394,-0.000380,-0.000546,-0.000737,-0.000885,-0.001113,-0.001202
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20240214__CC_taarray.ext:82]: spot checked looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20240214__CC_tcarray.ext [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+0.007039,0.006532,0.006339,0.006221,0.006538,0.006717,0.006008,0.005738,0.005506,0.005382,0.005048,0.004842,0.004691,0.004478,0.004096,0.003870,0.003501,0.003261,0.002576,0.002499,0.001919,0.001463,0.001099,0.000787,0.000246,0.000000,0.000056,-0.000148,-0.000314,-0.000323,-0.000318,-0.000344,-0.000247,-0.000133,-0.000181
+0.006517,0.006199,0.005853,0.005840,0.006308,0.006081,0.005652,0.005394,0.005110,0.004968,0.004724,0.004445,0.004363,0.004108,0.003876,0.003784,0.003317,0.002977,0.002539,0.002322,0.001705,0.001247,0.000939,0.000508,0.000279,0.000000,-0.000235,-0.000309,-0.000483,-0.000479,-0.000394,-0.000469,-0.000479,-0.000283,-0.000283
+0.006137,0.005915,0.005734,0.005747,0.005788,0.005878,0.005492,0.005266,0.004906,0.004790,0.004394,0.004354,0.004151,0.003977,0.003746,0.003273,0.003199,0.002726,0.002224,0.002114,0.001694,0.001296,0.001036,0.000503,0.000178,0.000000,-0.000303,-0.000204,-0.000563,-0.000529,-0.000396,-0.000490,-0.000588,-0.000340,-0.000381
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20240214__CC_tcarray.ext:82]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00151__20240221__CC_taarray.ext [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+-0.009704,-0.007472,-0.006745,-0.006194,-0.005870,-0.005638,-0.005292,-0.005097,-0.004626,-0.004070,-0.003605,-0.003495,-0.003154,-0.002890,-0.002441,-0.002461,-0.002058,-0.001687,-0.001482,-0.001108,-0.001055,-0.000785,-0.000354,-0.000177,0.000000,0.000281,0.000609,0.000519,0.000639,0.000980,0.000972,0.001027,0.000804,0.000937,0.000970
+-0.009649,-0.007455,-0.006894,-0.006307,-0.005874,-0.005654,-0.005354,-0.005120,-0.004600,-0.004020,-0.003591,-0.003405,-0.003206,-0.002941,-0.002468,-0.002404,-0.002076,-0.001603,-0.001377,-0.001026,-0.001042,-0.000755,-0.000402,-0.000235,0.000000,0.000304,0.000691,0.000517,0.000591,0.000978,0.000878,0.001076,0.000880,0.001043,0.000162
+-0.010459,-0.008181,-0.007563,-0.006989,-0.006493,-0.006196,-0.005900,-0.005609,-0.005220,-0.004436,-0.003893,-0.003765,-0.003511,-0.003245,-0.002746,-0.002653,-0.002285,-0.001929,-0.001545,-0.001214,-0.001218,-0.000926,-0.000538,-0.000286,0.000000,0.000302,0.000665,0.000403,0.000634,0.000773,0.000965,0.001053,0.000752,0.000786,0.000410
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00151__20240221__CC_taarray.ext:92]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00151__20240221__CC_tcarray.ext [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+0.000069,-0.000269,-0.000411,-0.000310,-0.000240,-0.000406,-0.000502,-0.000681,-0.000422,-0.000707,-0.000674,-0.000538,-0.000536,-0.000479,-0.000563,-0.000358,-0.000242,-0.000278,-0.000459,-0.000517,-0.000327,-0.000254,-0.000272,-0.000159,0.000000,-0.000113,-0.000135,0.000231,0.000197,0.000403,0.000363,0.000854,0.001083,0.001273,0.001482
+-0.000103,-0.000484,-0.000717,-0.000832,-0.000792,-0.000922,-0.001024,-0.000969,-0.000845,-0.000882,-0.001084,-0.000935,-0.000906,-0.000990,-0.000735,-0.000621,-0.000719,-0.000521,-0.000537,-0.000356,-0.000343,-0.000348,-0.000222,-0.000240,0.000000,0.000074,0.000120,0.000083,0.000283,0.000423,0.000398,0.000973,0.001063,0.001480,0.002058
+0.001727,0.001416,0.001240,0.001345,0.000811,0.000921,0.000109,0.000329,-0.000320,-0.000600,-0.000270,-0.000445,-0.000039,-0.000142,-0.000175,-0.000186,0.000153,-0.000003,-0.000212,-0.000259,-0.000229,-0.000203,-0.000247,0.000062,0.000000,-0.000182,-0.000237,0.000070,0.000327,0.000194,0.000101,0.000409,0.000396,0.000664,0.001865
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00151__20240221__CC_tcarray.ext:92]: spot checked, looks good

**Review by cooleyky — APPROVED** [review]
looks good

**Review by reedan88 — APPROVED** [review]
Looks good.

**Pull request merged**